fun-facts.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fun Facts</title>
+    <link rel="stylesheet" href="fun-facts.css">
+</head>
+
+<body>
+    <header>
+        <nav class="container">
+            <div class="main">
+                <div>
+                    <img class="logo" src="manta-logo-2.png" alt="Claudiu logo">
+                </div>
+                <ul>
+                    <li><a href="contact-page.html">Contact</a></li>
+                    <li class="active"><a href="fun-facts.html">Fun Facts</a>
+                    </li>
+                    <li><a href="claudiu-website-main-page.html">Home</a>
+                    </li>
+                </ul>
+
+            </div>
+        </nav>
+
+    </header>
+    <section>
+
+        <p id="main-text">Welcome and thanks for taking an interest and checking out my profile. I am currently
+            learning the ins
+            and
+            outs of front-end development but I am open to future collaborations. In the meantime, below are some
+            more
+            fun facts about me. Feel free to <a href="contact-page.html">click the contact </a>
+            button if you'd like to connect and stay in touch.</p>
+
+    </section>
+
+    <!-- List and Photos of Claudiu's Interests and Hobbies -->
+    <div class="clearfix">
+        <div class="image-container">
+            <img class="page-image"
+                src="https://images.unsplash.com/photo-1604398907654-ec2bf7108b4a?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8dHdvJTIwY2F0c3xlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=60"
+                alt="Cats" style="height: 200px;"><img class="page-image"
+                src="https://images.unsplash.com/photo-1503772582814-75e7ebd8a50c?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1778&q=80"
+                alt="Barcelona FC" style="height: 200px;"><img class="page-image"
+                src="https://images.unsplash.com/photo-1498050108023-c5249f4df085?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8Y29kaW5nfGVufDB8fDB8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=60"
+                alt="Computer Coding" style="height: 200px;">
+
+        </div>
```

Comment:
The images are currently given style attributes both inline in the html and on the css style sheet.  It is best practice to keep these in one place, (on the css style sheet), have a look into seperation of concerns.

When the page is viewed on a smaller screen, are the images behaving as you would expect?  Which of the 2 style instructions might you need to revisit?